backend/src/handlers/ridership.py

```diff
@@ -83,3 +128,33 @@ async def post_ridership_stats(req: Request, van_id: int):
         session.commit()
 
     return HardwareOKResponse()
+
+
+@router.get("/")
+def get_ridership(
+    req: Request, filters: Optional[RidershipFilterModel]
+) -> JSONResponse:
+    with req.app.state.db.session() as session:
+        if filters is None or filters.filters is None:
+            analytics: List[Analytics] = session.query(Analytics).all()
+        else:
+            analytics: List[Analytics] = (
+                session.query(Analytics).filter(*filters.filters).all()
+            )
+
+    # convert analytics to json
+    analytics_json = []
+    for analytic in analytics:
+        analytics_json.append(
+            {
+                "van_id": analytic.van_id,
```

Comment:
camelCase and use `latitude` and `longitude` instead of `lat`/`lon`. (Sorry, again this just makes life easier in JS)